lib/data_update_scripts/20201209134953_touch_articles_with_capitalized_paths.rb

```diff
@@ -0,0 +1,13 @@
+module DataUpdateScripts
+  class TouchArticlesWithCapitalizedPaths
+    def run
+      offending_articles = Article.where("path != lower(path)")
+
+      # Some log visibility for troubleshooting/monitoring
+      Rails.logger.info("Updating #{offending_articles.count} articles...")
+
+      # touch won't trigger the callbacks so a manual update will do the trick
+      offending_articles.update(updated_at: Time.current)
```

Comment:
Turns out `touch_all` doesn't run the `before_save` callbacks (only `before_touch` and others), which is how we're intending to fix the problem.

The idea is that if the logic is correct (fix introduced to `app/models/article.rb`) then this update and all subsequent (user triggered) will continue to work as expected.